### DISCUSSION TITLE
Limitation in the wiki page size we can save in the database.

### DISCUSSION THREAD

**Discussion by abockova-accenture** [requarks/wiki#6567] [Help / Questions] [answered]
My point is that there is a limitation in the wiki page size we can save in the database.

While editing the wiki page about azure resource logging, I got the attached error message.
![image](https://github.com/requarks/wiki/assets/91313783/709e1ccd-953c-4c0b-9e62-8568c6b081c4)

When looking at the payload during the save operation it reached around 8 Mb, please see attachment.
![image](https://github.com/requarks/wiki/assets/91313783/52e115ac-d4bd-4062-a899-0f871afb0d2c)
This was probably due to the fact that I added a lot of diagrams which are saved into the database.  This is a limitation we should not encounter if using storage to store diagrams and images rather than the database.

Could you help me with it?

**Comment by NGPixel** [comment, marked as answer]
That's not a database limit, but a body parser limit. You can increase its value in your config.yml:

https://github.com/Requarks/wiki/blob/main/config.sample.yml#L147

**Reply by abockova-accenture** [reply]
Thank you for your answer, but I`m not able to fix it, because we are using container image requarks/wiki:2. Could it be solved in the next version, or fixed in the older as bag fix?

**Reply by NGPixel** [reply]
There is nothing to fix, this is by design. You can mount a custom config file when using docker:

https://docs.requarks.io/install/docker#alternative-mount-the-config-file
